instantmusic-0.1/bin/instantmusic

```diff
@@ -114,7 +114,7 @@ def query_and_download(search, has_prompts=True, is_quiet=False):
     # We only ask the user which one they want if prompts are on, of course
     if has_prompts and not is_quiet:
         choice = ''
-        while choice.strip() == '':
+        while not(choice.isdigit()) or not(0 <= int(choice) < len(available)):
```

Comment:
```
while not(choice.isdigit()) or not(0 <= int(choice) < len(available)):
    print("Oups, that was wrong. Try again!")
```

We come inside the loop if the input is valid., then why respond by `print("Oups, that was wrong. Try again!")` ?